index.html

```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="actions">
+              <button class="action action-text btn">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-3.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">TONIC</p>
+              <ul class="client-role-year">
+                <li class="client">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+              </ul>
+            </div>
+            <p class="portfolio-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="actions">
+              <button class="action action-text btn">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-1.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">Multi-Post Stories</p>
+              <ul class="client-role-year">
+                <li class="client">CANOPY</li>
+                <li class="role">Back End Dev</li>
+                <li class="year">2015</li>
+              </ul>
+            </div>
+            <p class="portfolio-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="actions">
+              <button class="action action-text btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="top">
+          <p class="about-my-self-title">About Myself</p>
```

Comment:
- [ ] for best SEO performance, the title of the sections (e.g., "About Myself") should use a header tag `hx` (h1..h6) so kindly fix it ,  you can read more about it [here](https://www.reliablesoft.net/h1-tag/)